cmd/kube-controller-manager/app/options/options.go

```diff
@@ -193,6 +194,12 @@ func NewKubeControllerManagerOptions() *KubeControllerManagerOptions {
 
 	s.GarbageCollectorController.GCIgnoredResources = gcIgnoredResources
 
+	rqIgnoredResources := make([]componentconfig.GroupResource, 0, len(quotainstall.DefaultIgnoredResources()))
```

Comment:
assign `quotainstall.DefaultIgnoredResources()` to a var. Calling it twice looks fishy.

Reply:
Done. Also rebased on top of master.